### ISSUE TITLE
HTML report: Fix 'Passed core (Can be certified)' result

### ISSUE THREAD

**Issue by dstenger** [opengeospatial/teamengine#248]
When reference implementation [1] of ets-wfs20 is tested via Web Browser Interface, the ```Passed core (Can be certified)``` result says ```No``` although the service passes all basic tests (also add ```Featureidentifier: PS_PROTECTEDSITE_a367af0b-9457-4445-9cbd-eb48ae7a844a``` when starting the test).

@keshav-nangare Can you please investigate why the result is wrong?

Further, there should be a space added to text:
* Current status: ```Passed core(Can be certified): No```
* Correct status: ```Passed core (Can be certified): No```

[1] http://cite.deegree.org/deegree-webservices-3.4-RC3/services/wfs200?service=WFS&request=GetCapabilities

**Comment by keshav-nangare** [comment]
@dstenger 

Maybe we need to add configuration of conformance class in TestNG test then it will show something.

Here is the config link: https://github.com/opengeospatial/cite/wiki/Encoding-Conformance-Classes

In above link there is example to add basic conformance classes.

**Comment by dstenger** [comment]
Thank you. I added following to the ets-wfs20:
```
...
      <suite>
        <namespace-uri>http://www.opengis.net/cite/iso19142</namespace-uri>
        <prefix>tns</prefix>
        <local-name>ets-${ets-code}-${version}</local-name>
        <title>WFS 2.0 (ISO 19142:2010) Conformance Test Suite</title>
        <description>Checks WFS 2.0 implementations for conformance to ISO 19142:2010.</description>
        <BasicConformanceClasses>
          <conformanceClass>Preconditions</conformanceClass>
          <conformanceClass>All GML application schemas</conformanceClass>
          <conformanceClass>GML application schemas defining features</conformanceClass>
          <conformanceClass>Simple WFS</conformanceClass>
          <conformanceClass>Basic WFS</conformanceClass>
        </BasicConformanceClasses>
      </suite>
...
```

However, the test suite now always states following:
```
Passed core(Can be certified): Yes
```
This is even the case if not all core tests are passed.

@keshav-nangare Can you please take a look at it?

**Comment by keshav-nangare** [comment]
Now the HTML report will display correct result for core test.

If one of the core test is failed then it will set to **`'No'`** otherwise **`'Yes'`**.

Fixed with this commit: 1b734894e24f68eb58ce8adaecd7d154f053cdfb

**Comment by dstenger** [comment]
Fix was successfully tested with [1] and changes from https://github.com/opengeospatial/teamengine/issues/248#issuecomment-333505626 via TEAM Engine Web Interface.

If Featureidentifier ```PS_PROTECTEDSITE_a367af0b-9457-4445-9cbd-eb48ae7a844a``` is passed, report says ```Passed core (Can be certified): Yes```.
If no Featureidentifier is passed, report says ```Passed core (Can be certified): No```.

--> Expected behaviour.

[1] http://cite.deegree.org/deegree-webservices-3.4-RC3/services/wfs200?service=WFS&request=GetCapabilities
